src/common/libpmi/pmi.c

```diff
@@ -324,7 +324,7 @@ int PMI_Spawn_multiple(int count,
 
 int PMI_Get_clique_ranks (int ranks[], int length)
 {
-    int result;
+    int result = PMI_ERR_INIT;
```

Comment:
rebase / merge issue?  It looks like you've added a useless initialization?

Reply:
Uh oh.  Thanks!